src/compile.rs

```diff
@@ -36,15 +55,26 @@ pub fn run(
 }
 
 fn handle_module(
+    _flag_verbose: bool,
```

Comment:
Please remove leading `_`.

Reply:
When I initially added this code, I got a warning from cargo suggesting that I add the leading `_`. I've removed it and it looks like it's working fine now, though.